<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_Normal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Clignotant">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clignotant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Automatique">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Automatique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Test_Feux">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TEST Feux</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="te_2Fin">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>321</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="te_3Fin">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_feux1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_feux2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>91</width>
      <height>161</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_Info_plage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Plages Horaires - Mode Automatique</string>
    </property>
    <widget class="QTimeEdit" name="te_4Debut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="te_4Fin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_info_fin">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Fin</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="te_1Fin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="te_1Debut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
    <widget class="QLabel" name="l_info_debut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>DÃ©but</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="te_2Debut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="te_3Debut">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="l_DateHeure">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>               DATE / HEURE</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_feu_1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Feu 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_feu_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Feu 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Feux Tricolores</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_F1Rouge">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>R.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_F1Vert">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>V.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_F2Rouge">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>R.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_F1Orange">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>O.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_F2Orange">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>O.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="l_F2Vert">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>V.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>gb_Info_plage</zorder>
   <zorder>pb_Normal</zorder>
   <zorder>pb_Clignotant</zorder>
   <zorder>pb_Automatique</zorder>
   <zorder>pb_Test_Feux</zorder>
   <zorder>te_2Fin</zorder>
   <zorder>te_3Fin</zorder>
   <zorder>te_feux1</zorder>
   <zorder>te_feux2</zorder>
   <zorder>l_DateHeure</zorder>
   <zorder>l_feu_1</zorder>
   <zorder>l_feu_2</zorder>
   <zorder>label</zorder>
   <zorder>l_F1Rouge</zorder>
   <zorder>l_F1Vert</zorder>
   <zorder>l_F2Rouge</zorder>
   <zorder>l_F1Orange</zorder>
   <zorder>l_F2Orange</zorder>
   <zorder>l_F2Vert</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
